Modules/Net/popup.swift

```diff
@@ -260,6 +260,15 @@ internal class Popup: NSStackView, Popup_p {
         })
     }
     
+    private func flag(country:String) -> String {
+        let base : UInt32 = 127397
+        var s = ""
+        for v in country.unicodeScalars {
+            s.unicodeScalars.append(UnicodeScalar(base + v.value)!)
```

Comment:
Hi. Are you sure that `UnicodeScalar` cannot be nil?

Reply:
Cannot be nil. But can be no flag symbol. Closed

Reply:
[UnicodeScalar](https://developer.apple.com/documentation/swift/unicode/scalar/2905753-init):
`If v is an invalid Unicode scalar value, the result is nil.`